### PR TITLE
Edits for punctuation, usage, and clarity

### PR DESCRIPTION
**Pull request by misterpantz**

### CODE REVIEW

File: docs/user-manual.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
...
-Follow this step to set up the Mondoo operator using `kubectl` and a manifest file.
+Follow this step to set up the Mondoo operator using kubectl and a manifest file.
 
-__Precondition:__ `kubectl` with cluster admin access
+Precondition: kubectl with cluster admin access
 
-__->__ Apply the operator manifests:
+&rarr; Apply the operator manifests:
 
     ```bash
```
[the remark this answers is not in the document]
Comment (misterpantz) [reply docs/user-manual.md]: I fixed the markdown by removing the indent. Thanks for catching that!

File: docs/user-manual.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
-    ```
+```bash
+curl -sSL https://github.com/mondoohq/mondoo-operator/releases/latest/download/mondoo-operator-manifests.yaml > mondoo-operator-manifests.yaml
+kubectl apply -f mondoo-operator-manifests.yaml
+```
 
-### Installation with helm
-The following steps setup a development Kubernetes to test the operator using [helm](https://helm.sh/).
+### Installing with Helm
+Follow these steps to set up a development Kubernetes to test the operator using [Helm](https://helm.sh/).
```
Comment (joelddiaz) [inline docs/user-manual.md:26]:
```suggestion
Follow these steps to set up the Mondoo Operator using [Helm](https://helm.sh/).
```

File: docs/user-manual.md [hunk 3]
```diff
@@ -83,13 +83,13 @@ To configure the Mondoo client secret to the following steps:
...
 
-3. Store service account as a secret in the mondoo namespace via:
+3. Store the service account as a secret in the Mondoo namespace:
     ```bash
     kubectl create secret generic mondoo-client --namespace mondoo-operator --from-file=config=creds.json
     ```
 
-## Creating the MondooAuditConfig
-Once the secret is configured, we configure the operator to define the scan targets:
+## Creating MondooAuditConfig
```
Comment (joelddiaz) [inline docs/user-manual.md:91]: MondooAuditConfig is a type of object (of which there can be more than one). In the same way we would say "first create a Namespace" or "update the contents of the Secret", it makes sense to me to talk about "creating a MondooAuditConfig".
```suggestion
## Creating a MondooAuditConfig
```

File: docs/user-manual.md [hunk 4]
```diff
@@ -107,32 +107,35 @@ Once the secret is configured, we configure the operator to define the scan targ
...
+- Install and use cert-manager to automate the creation and update of the TLS certs
+- Use the OpenShift certificate creation/rotation features
+- Create (and rotate) your own TLS certificates manually
 
-A working setup should show the Pods being created/modified/deleted being processed by the webhook Pod.
+A working setup shows the webhook pod processing the created/modified/deleted pods.
 
 Display the logs in one window:
 ```bash
 kubectl logs -f deployment/mondoo-operator/mondoo-operator-webhook-manager -n mondoo-operator
```
Comment (joelddiaz) [inline docs/user-manual.md:127]: The names of objects have subtly changed recently
```suggestion
kubectl logs -f deployment/mondoo-client-webhook-manager -n mondoo-operator
```

File: docs/user-manual.md [hunk 5]
```diff
@@ -107,32 +107,35 @@ Once the secret is configured, we configure the operator to define the scan targ
...
 ```
 
-And create/modify/delete a Pod in another window:
+And create/modify/delete a pod in another window:
```
Comment (joelddiaz) [inline docs/user-manual.md:130]: `Pod` is the proper name of a specific type of object in k8s. In the same way that we refer to Deployments and ConfigMaps and Secrets, a Pod is also one of these "special named objects". See the k8s docs: https://kubernetes.io/docs/concepts/workloads/pods/

File: docs/user-manual.md [hunk 6]
```diff
@@ -154,27 +157,27 @@ The easiest way to bootstrap the admission controller TLS certificate is by usin
           mode: cert-manager
     ```
 
-3. The admission controller `Deployment` should start up, and the `ValidatingWebhookConfiguration` should be annotated to insert the Certificate Authority data along with the Secret containing the TLS certs will be created by cert-manager (named webhook-serving-cert).
+The admission controller `Deployment` should start. `ValidatingWebhookConfiguration` should be annotated to insert the certificate authority data. cert-manager creates a secret named `webhook-serving-cert` that contains the TLS certificates.
```
Comment (joelddiaz) [inline docs/user-manual.md:160]:
```suggestion
The admission controller `Deployment` should start.  The`ValidatingWebhookConfiguration` should be annotated to insert the certificate authority data. cert-manager creates a secret named `webhook-serving-cert` that contains the TLS certificates.
```

File: docs/user-manual.md [hunk 7]
```diff
@@ -154,27 +157,27 @@ The easiest way to bootstrap the admission controller TLS certificate is by usin
...
    ```
 
-2. Generate certificate for your CA.
+2. Generate a certificate for your certificate authority:
    ```bash
    openssl req -x509 -new -nodes -key ca.key -subj "/CN=Webhook Issuer" -days 10000 -out ca.crt
    ```
 
-3. Generate key for webhook server pod.
+3. Generate a key for the webhook server pod:
```
Comment (joelddiaz) [inline docs/user-manual.md:175]:
```suggestion
3. Generate a key for the webhook server Pod:
```

File: docs/user-manual.md [hunk 8]
```diff
@@ -202,28 +205,28 @@ It is possible to manually create the TLS certificate required for the admission
...
    ```
 
-6. Generate the certificate for the webhook server.
+6. Generate the certificate for the webhook server:
    ```bash
    openssl x509 -req -in server.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out server.crt -days 10000 -extensions v3_ext -extfile csr.conf
    ```
 
-7. Create the Secret holding the TLS certificate data.
+7. Create the secret holding the TLS certificate data:
```
Comment (joelddiaz) [inline docs/user-manual.md:218]: A Secret is the proper name of a specific type of k8s resource/object. https://kubernetes.io/docs/concepts/configuration/secret/
```suggestion
7. Create the Secret holding the TLS certificate data:
```

File: docs/user-manual.md (hunk 5 above)
Comment (misterpantz) [reply docs/user-manual.md:130]: I'm reading their docs... their choices strike me as inconsistent. But I agree we should follow them.

**Comment by imilchev** [comment]
I assume that the changes in this PR were also present in #342 which just got merged. I will close this now

**Pull request closed without merging**